src/schemas/listing-core.json

```diff
@@ -0,0 +1,243 @@
+{
+  "$id": "http://schema.originprotocol.com/listing-core-v1.0.0",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Origin Protocol core listing schema",
+  "definitions": {
+    "moneyAmount": {
+      "type": "object",
+      "title": "An amount of money in a specified currency",
+      "properties": {
+        "currency": {
+          "type": "string"
+        },
+        "amount": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "currency",
+        "amount"
+      ]
+    },
+    "location": {
+      "type": "object",
+      "properties": {
+        "street": {
+          "type": "string"
+        },
+        "extendedStreet": {
+          "type": "string"
+        },
+        "locality": {
+          "type": "string"
+        },
+        "region": {
```

Comment:
We might need more fields here? It is hard to fit all kinds of international addressing systems into one schema. We might need more than just region. Google deals with this by using
`administrative_area_level_1`,... `administrative_area_level_5`. E.g. in the US:

- `administrative_area_level_1` corresponds to a state
- `administrative_area_level_2` corresponds to a county

But in other countries, they can just be adapted to whatever administrative divisions there are. I don't know too much about other addressing systems, but in places like Japan there might be quite a few administrative divisions, e.g. prefectures, districts, cities, wards.

Reply:
@franckc are we following (aka copying) some other standard on location?

Meta comment: I'm still a little perplexed by our strategy of not wanting to attempt doing listing schemas until we can "do them right" (and fear of getting locked into a bad standard), but then OTOH now including a location standard—something notoriously hard to "do right"!

Reply:
Good feedback @tomlinton and @wanderingstan This was primarily modeled after schema.org. Location is out of scope for V1 anyway so let me just take it out of this schema. We need more research/thinking - I agree better to not put it our there if we are not completely sure it is the right schema... :)